### PR TITLE
BC-234 Support PV Schedule Update for send_signed_change_request management command.

### PR DESCRIPTION
**Pull request by olegtropinin**

### CODE REVIEW

File: node/blockchain/facade.py [hunk 1]
```diff
@@ -143,9 +143,23 @@ def get_next_block_identifier(self) -> Optional[BlockIdentifier]:
...
+        return AccountState(
+            _id=account_number,
+            balance=0,
+            account_lock=account_number,
+            node=None,
+        )
+
+    @staticmethod
+    def get_account_state(account_number) -> ORMAccountState:
+        return ORMAccountState.objects.get_or_none(_id=account_number
```
Comment (dmugtasimov) [inline node/blockchain/facade.py:157]: why do not you use `update_or_create()`?

File: node/blockchain/facade.py [hunk 2]
```diff
@@ -143,9 +143,23 @@ def get_next_block_identifier(self) -> Optional[BlockIdentifier]:
...
+                                                   ) or BlockchainFacade.create_account_state(account_number)
+
     @staticmethod
     def get_account_lock(account_number) -> AccountLock:
-        account_state = ORMAccountState.objects.get_or_none(_id=account_number)
+        account_state = BlockchainFacade.get_account_state(account_number)
```
Comment (dmugtasimov) [inline node/blockchain/facade.py:162]: I do not think it is a good idea to create empty account states

File: node/blockchain/management/commands/send_signed_change_request.py [hunk 3]
```diff
@@ -1,24 +1,43 @@
...
 
+LOCAL_BLOCKCHAIN = 'local'
+
 
 def add_common_args(parser):
-    parser.add_argument('node-address', type=str)
+    parser.add_argument(
+        'node-address',
+        type=str,
+        help=f'"Domain/IP-address" of remote Node. Or {LOCAL_BLOCKCHAIN} '
```
Comment (dmugtasimov) [inline node/blockchain/management/commands/send_signed_change_request.py:24]: `help='node address or "local" to denote local blockchain operation"

File: node/blockchain/views/account_state.py [hunk 4]
```diff
@@ -27,10 +27,4 @@ def get_object(self):
...
 
             # If account is not found then we virtually assume that it exists (although not know to the blockchain)
-            # TODO(dmu) MEDIUM: Should we move this logic to AccountStateManage or BlockchainFacade?
-            return AccountState(
-                _id=account_number,
-                balance=0,
-                account_lock=account_number,
-                node=None,
-            )
+            return BlockchainFacade.create_account_state(account_number)
```
Comment (dmugtasimov) [inline node/blockchain/views/account_state.py:30]: we should actually create the account state on GET-requests

File: node/blockchain/facade.py (hunk 1 above)
Comment (olegtropinin) [reply node/blockchain/facade.py:157]: Do you mean get_or_create()?

File: node/blockchain/facade.py (hunk 2 above)
Comment (olegtropinin) [reply node/blockchain/facade.py:162]: It just created empty AccountState object. It didn't save this object in database.

**Pull request merged**